onnxruntime/python/tools/quantization/E2E_example_model/object_detection/trt/yolov3/evaluate.py

```diff
@@ -10,6 +10,8 @@
 from onnxruntime.quantization.calibrate import CalibrationDataReader
 import numpy as np
 
+import torch
```

Comment:
are we able to implement this without torch dependency? 
looks like it's mainly for NMS and a couple other operations? 
add it as a TODO?

Reply:
I think we can implement operations without torch, but it takes time to investigate and implement.
Will do it later.